src/mono/msvc/libmonoruntime-common.targets

```diff
@@ -73,6 +73,7 @@
     <ClCompile Include="$(MonoSourceLocation)\mono\metadata\marshal-ilgen.c" />
     <ClInclude Include="$(MonoSourceLocation)\mono\metadata\marshal-ilgen.h" />
     <ClInclude Include="$(MonoSourceLocation)\mono\metadata\marshal-internals.h" />
+    <ClCompile Include="$(MonoSourceLocation)\mono\metadata\memory-manager.c" />
     <ClCompile Include="$(MonoSourceLocation)\mono\metadata\mempool.c" />
```

Comment:
Should include memory-manager.h as well.

Reply:
I was just using the loader-internals header for all this. Though maybe I should just make a separate header...